src/wcs2_0/src/main/java/org/geoserver/wcs2_0/response/MultidimensionalSupportingCoverageResponse.java

```diff
@@ -0,0 +1,12 @@
+/* (c) 2021 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.wcs2_0.response;
+
+/**
+ * Marker interface to be implemented by {@link
+ * org.geoserver.wcs.responses.CoverageResponseDelegate} implementations supporting Multidimensional
+ * format
+ */
+public interface MultidimensionalSupportingCoverageResponse {}
```

Comment:
Can it be a bit shorter?

```suggestion
public interface MultiDimensionalCoverageResponse {}
```

Reply:
Updated. thanks